src/panels/lovelace/cards/hui-gauge-card.ts

```diff
@@ -218,6 +220,40 @@ class HuiGaugeCard extends LitElement implements LovelaceCard {
     fireEvent(this, "hass-more-info", { entityId: this._config!.entity });
   }
 
+  private _attachObserver(): void {
+    if (typeof ResizeObserver !== "function") {
+      import("resize-observer").then((modules) => {
+        modules.install();
+        this._attachObserver();
+      });
+      return;
+    }
```

Comment:
we should create a helper for this by now

Reply:
I was thinking the same when I copied and pasted.